data/campaigns/The_South_Guard/scenarios/03_A_Desparate_Errand.cfg

```diff
@@ -392,54 +412,109 @@
+            message= _ "Stay your hand, Linderion. This man bears the banner of Wesnoth and appears to come to us in peace. Tell us, what brings you to the hallowed citadel of Elrath?"
         [/message]
 
         [message]
             speaker=Deoran
-            message= _ "My lady, we have come to the elves to seek guidance. Bandits and criminals in our land have summoned the dead to help them plunder our farms and villages. They are coming across the river from the great forest to the south of our lands, where no man has yet traveled."
+            message= _ "My lady, we have come to the elves to seek guidance. Bandits and criminals have summoned the dead to help them plunder our farms and villages. They come from the great forest across the river to our south, where no man has yet traveled. We have defeated many a bandit already, but I fear that we cannot continue to battle them for much longer with the undead under their control."
         [/message]
 
         [message]
             speaker=Ethiliel
-            message= _ "The elves know of the great southern forest, and have walked its paths many times. What would you do, Deoran?"
+            message= _ "The elves know of the great southern forest, and have walked its paths many times."
         [/message]
 
         [message]
             speaker=Deoran
-            message= _ "I would lead my men south of the river to find the source of these beings of darkness that we might destroy it. For surely, ere that be done, we cannot stand against them for long."
+            message= _ "And what do you know of the undead?"
         [/message]
 
         [message]
             speaker=Ethiliel
-            message= _ "Indeed you are brave to face such foes. But I fear that alone you will fail and fall. One elf now lives who remembers the secrets of the undead that we learned when Haldric I came to our land. His name is Mebrin, and he lives, secluded in the hills, near here. I will take you to him."
+            message= _ "My answer will depend on your motives, young human. What would you do, knowing the secrets of the unholy ones?"
+        [/message]
+
+        [message]
+            speaker=Deoran
+            message= _ "I would lead my men south of the river to find and destroy the source of these beings of darkness."
+        [/message]
+
+        [message]
+            speaker=Vardanos
+            message= _ "A fanciful delusion! A human untrained in the mystic arts stands no chance against the undead, especially not one so young!"
+        [/message]
+
+        [message]
+            speaker=Deoran
+            message= _ "I have proven myself capable of defending my land and leading my men to victory—"
+        [/message]
+
+        [message]
+            speaker=Eltenmir
+            message= _ "In battles against other <i>humans</i>, young commander! To fight such a profane evil is another matter altogether."
```

Comment:
This is the third dialogue line in a row from the elves that refers to Deoran as "young". I think some variance would be good. Deoran has also fought and defeated undead up to this point, they've just been weak. Perhaps here have "In battles against <i>other humans</i> and feeble minions. To bring war to the source of such unnatural evil is another matter altogether."

Reply:
Perhaps"fell" would be a good word choice instead of "unnatural" or "profane".